.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on: push
+jobs:
+  CI:
+    name: Continuous Integration
+    runs-on: ubuntu-latest
+    permissions:
+      # required by guardian/actions-assume-aws-role
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: guardian/actions-assume-aws-role@v1.0.0
+        with:
+          awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

Comment:
https://github.com/actions/setup-node/pull/338 allows you to specify a `node-version-file`, which makes it easier to keep things inline.

Reply:
Thanks! Even better 😃